eng/pipelines/preview.yml

```diff
@@ -68,6 +68,13 @@ extends:
         $newContent | Set-Content $path -NoNewline
       displayName: 'Update Packages.Data.props'
       workingDirectory: $(sdkRepositoryPath)
+    AutorestInitializationSteps:
+    - ${{ if eq(parameters.UseTypeSpecNext, 'true') }}:
+      - task: Bash@3
+        inputs:
+          targetType: 'inline'
+          script: 'echo "##vso[build.addbuildtag]typespec_next"'
```

Comment:
so the magic bash script can add a filter in the UI? Is this the formal way to do so?

Reply:
> so the magic bash script can add a filter in the UI?

Yes.

> Is this the formal way to do so?

To be honest I don't know the answer. But we don't have other ways.

Reply:
@hallipr Could you help review this PR?

Reply:
This is the standard way to add tags to a build mid-run.  See:

[Scheduled Builds](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Core_1.40.0/eng/pipelines/templates/jobs/ci.yml#L72)

[Release Tags](
https://github.com/Azure/azure-sdk-for-net/blob/Azure.Core_1.40.0/eng/pipelines/templates/jobs/ci.mgmt.yml#L54)